src/sysext/sysext.c

```diff
@@ -948,6 +957,7 @@ static int verb_help(int argc, char **argv, void *userdata) {
                "     --force              Ignore version incompatibilities\n"
                "     --image-policy=POLICY\n"
                "                          Specify disk image dissection policy\n"
+               "     --noexec=BOOL        Whether to mount confexts with noexec\n"
```

Comment:
hmm, it's not the confexts that are mounted like that, but the resulting overlay stack